webapp/store/views.py

```diff
@@ -115,21 +115,7 @@ def search():
 
 @jaasstore.route("/u/<username>")
 def user_details(username):
-    entities = models.get_user_entities(username)
-    if len(entities["charms"]) > 0 or len(entities["bundles"]) > 0:
-        return render_template(
-            "store/user-details.html",
-            context={
-                "bundles_count": len(entities["bundles"]),
-                "bundles": entities["bundles"],
-                "charms_count": len(entities["charms"]),
-                "charms": entities["charms"],
-                "entities": entities,
-                "username": username,
-            },
-        )
-    else:
-        return abort(404, "User not found: {}".format(username))
+    return abort(404, "User not found: {}".format(username))
```

Comment:
Should this maybe be something more descriptive as right now it presents as a bug to the user if they have previously logged in before this goes out.

Reply:
We could but this is would be something we need to be a UI we need to support going forwards instead of moving to a place where we have chosen to be. What would you recommend? Removed error with a note?

Reply:
Done, what you think?

Reply:
Thanks much better :D